src/components/Accordion/Accordion.tsx

```diff
@@ -61,20 +62,26 @@ export const Accordion = (props: AccordionProps): React.ReactElement => {
     'usa-accordion',
     {
       'usa-accordion--bordered': bordered,
+      'aria-multiselectable': multiselectable,
     },
     className
   )
 
   const toggleItem = (itemId: AccordionItem['id']): void => {
-    const newOpenItems = [...openItems]
```

Comment:
Sooo ... it turns out that `const` in TypeScript just means you cannot reassign the variable. You can still invoke methods that change the object.

Here's a [playground](https://www.typescriptlang.org/play?#code/MYewdgzgLgBKlQIICdkEMCeMC8MDaAugNwBQ80K6GAdAA4CuEAFgBQCMAlKeSADYCm1XiADmLcklSYuQA) that shows adding an element to a `const` array.